version.php

```diff
@@ -27,7 +27,7 @@
 defined('MOODLE_INTERNAL') || die();
 
 
-$plugin->version   = 2020092500;        // The current plugin version (Date: YYYYMMDDXX)
+$plugin->version   = 2020092501;        // The current plugin version (Date: YYYYMMDDXX)
 $plugin->release   = 2020092500;        // The current plugin version (Date: YYYYMMDDXX)
```

Comment:
Hye Jamie,

Can you please update both values to be the same? And we probably should use `2020093000` as the version number.

If you can please update that and the matching number in `upgrade.php` it should be good to go

Thanks,
Jay